src/db_options.rs

```diff
@@ -74,6 +74,94 @@ impl Drop for Cache {
     }
 }
 
+pub struct Env {
```

Comment:
Please add a doc-comment for this struct.

Reply:
I added description for `Env`. Thank you for quick reviewing :+1: @DarkEld3r